Formula/arangodb.rb

```diff
@@ -23,41 +26,91 @@ class Arangodb < Formula
   end
 
   def install
-    # clang on 10.8 will still try to build against libstdc++,
-    # which fails because it doesn't have the C++0x features
-    # arangodb requires.
     ENV.libcxx
 
-    args = %W[
-      --disable-dependency-tracking
-      --prefix=#{prefix}
-      --disable-relative
-      --datadir=#{share}
-      --localstatedir=#{var}
-    ]
+    if build.head?
+      args = %W[
+        -DVERBOSE=On
+        -DHOMEBREW=On
+        -DUSE_OPTIMIZE_FOR_ARCHITECTURE=Off
+        -DASM_OPTIMIZATIONS=Off
+        -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
Normally that allows people to debug crashes without compromising performance. But with std_cmake_args I assume it should not specified at all.